Serverless: Packaging service...
Serverless: Excluding development dependencies...
Serverless: Creating Stack...
Serverless: Checking Stack create progress...
.....
Serverless: Stack create finished...
Serverless: Uploading CloudFormation file to S3...
Serverless: Uploading artifacts...
Serverless: Uploading service realworld.zip file to S3 (27.96 MB)...
Serverless: Validating template...
Serverless: Updating Stack...
Serverless: Checking Stack update progress...
........................................................................................................................................................................................................................................................................................................................................................................................................................................................................
Serverless: Stack update finished...
Service Information
service: realworld
stage: dev
region: eu-west-1
stack: realworld-dev
resources: 100
api keys:
  None
endpoints:
  POST - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/users
  POST - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/users/login
  GET - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/user
  PUT - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/user
  GET - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/profiles/{username}
  POST - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/profiles/{username}/follow
  DELETE - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/profiles/{username}/follow
  POST - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/articles
  GET - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/articles/{slug}
  PUT - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/articles/{slug}
  DELETE - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/articles/{slug}
  POST - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/articles/{slug}/favorite
  DELETE - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/articles/{slug}/favorite
  GET - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/articles
  GET - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/articles/feed
  GET - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/tags
  POST - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/articles/{slug}/comments
  GET - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/articles/{slug}/comments
  DELETE - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/articles/{slug}/comments/{id}
  GET - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/ping
  DELETE - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/__TESTUTILS__/purge
  POST - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/consent
  DELETE - https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api/consent
functions:
  createUser: realworld-dev-createUser
  loginUser: realworld-dev-loginUser
  getUser: realworld-dev-getUser
  updateUser: realworld-dev-updateUser
  getProfile: realworld-dev-getProfile
  followUser: realworld-dev-followUser
  unfollowUser: realworld-dev-unfollowUser
  createArticle: realworld-dev-createArticle
  getArticle: realworld-dev-getArticle
  udpateArticle: realworld-dev-udpateArticle
  deleteArticle: realworld-dev-deleteArticle
  favoriteArticle: realworld-dev-favoriteArticle
  unfavoriteArticle: realworld-dev-unfavoriteArticle
  listArticles: realworld-dev-listArticles
  getArticlesFeed: realworld-dev-getArticlesFeed
  getTags: realworld-dev-getTags
  createComment: realworld-dev-createComment
  getComments: realworld-dev-getComments
  deleteComment: realworld-dev-deleteComment
  ping: realworld-dev-ping
  purgeData: realworld-dev-purgeData
  consent: realworld-dev-consent
  revoke: realworld-dev-revoke
  watchtower-ingestion: realworld-dev-watchtower-ingestion
  watchtower-monitor: realworld-dev-watchtower-monitor
  wt-monitor-trigger: realworld-dev-wt-monitor-trigger
layers:
  None
Serverless: Run the "serverless" command to setup monitoring, troubleshooting and testing.

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  150 (ERR)
    1) "before all" hook in "Article"


  0 passing (161ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_56_51_151Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  147 (ERR)
    1) "before all" hook in "Article"


  0 passing (158ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_57_03_472Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  62 (ERR)
    1) "before all" hook in "Article"


  0 passing (74ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_57_13_628Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  58 (ERR)
    1) "before all" hook in "Article"


  0 passing (68ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_57_23_964Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  59 (ERR)
    1) "before all" hook in "Article"


  0 passing (70ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_57_34_823Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  84 (ERR)
    1) "before all" hook in "Article"


  0 passing (94ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_57_45_904Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  104 (ERR)
    1) "before all" hook in "Article"


  0 passing (114ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_57_56_241Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  75 (ERR)
    1) "before all" hook in "Article"


  0 passing (87ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_58_06_313Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  465 (ERR)
    1) "before all" hook in "Article"


  0 passing (476ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_58_17_003Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  60 (ERR)
    1) "before all" hook in "Article"


  0 passing (71ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_58_27_382Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  60 (ERR)
    1) "before all" hook in "Article"


  0 passing (71ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_58_37_889Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  61 (ERR)
    1) "before all" hook in "Article"


  0 passing (71ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_58_48_281Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  76 (ERR)
    1) "before all" hook in "Article"


  0 passing (87ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_58_58_620Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  65 (ERR)
    1) "before all" hook in "Article"


  0 passing (75ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_59_08_942Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  62 (ERR)
    1) "before all" hook in "Article"


  0 passing (73ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_59_19_522Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  62 (ERR)
    1) "before all" hook in "Article"


  0 passing (72ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_59_30_551Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  59 (ERR)
    1) "before all" hook in "Article"


  0 passing (71ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_59_40_782Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  57 (ERR)
    1) "before all" hook in "Article"


  0 passing (70ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T12_59_51_153Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  87 (ERR)
    1) "before all" hook in "Article"


  0 passing (97ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T13_00_01_306Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  65 (ERR)
    1) "before all" hook in "Article"


  0 passing (77ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T13_00_11_558Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  75 (ERR)
    1) "before all" hook in "Article"


  0 passing (85ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T13_00_22_593Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  127 (ERR)
    1) "before all" hook in "Article"


  0 passing (139ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T13_00_32_827Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  61 (ERR)
    1) "before all" hook in "Article"


  0 passing (71ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T13_00_42_640Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  81 (ERR)
    1) "before all" hook in "Article"


  0 passing (92ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T13_00_52_994Z-debug.log

> realworld-dynamodb-lambda@1.0.0 test:deployed /home/ubuntu/realworld-dynamodb-lambda
> API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha



Testing API_URL: [https://mb493pvsle.execute-api.eu-west-1.amazonaws.com/dev/api]
Done!

  Article
**** Call ## 0 ## Duration:  115 (ERR)
    1) "before all" hook in "Article"


  0 passing (127ms)
  1 failing

  1) Article
       "before all" hook in "Article":
     Error: Request failed with status code 500
      at createError (node_modules/axios/lib/core/createError.js:16:15)
      at settle (node_modules/axios/lib/core/settle.js:17:12)
      at IncomingMessage.handleStreamEnd (node_modules/axios/lib/adapters/http.js:236:11)
      at endReadableNT (_stream_readable.js:1064:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realworld-dynamodb-lambda@1.0.0 test:deployed: `API_URL=`serverless info --verbose | grep '^ServiceEndpoint:' | grep -o 'https://.*'`; API_URL=$API_URL/api mocha`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realworld-dynamodb-lambda@1.0.0 test:deployed script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-05-06T13_01_03_802Z-debug.log
results20200506125159/rec-res-9-props/
Serverless: Getting all objects in S3 bucket...
Serverless: Removing objects in S3 bucket...
Serverless: Removing Stack...
Serverless: Checking Stack removal progress...
....................................................................................................................................................................................................................................................................................................................................
Serverless: Stack removal finished...
